test/jasmine/tests/click_test.js

```diff
@@ -180,7 +180,9 @@ describe('Test click interactions:', function() {
 
     describe('drag interactions', function() {
         beforeEach(function(done) {
-            Plotly.plot(gd, mockCopy.data, mockCopy.layout).then(done);
+            Plotly.plot(gd, mockCopy.data, mockCopy.layout).then(function() {
+                setTimeout(done, 200);
```

Comment:
I thought Firefox wasn't affected by #445. I will adapt #445 to Firefox and see if the test works without the pause.